lib/api/2.0/notification.js

```diff
@@ -12,6 +12,31 @@ var notificationPost = controller({success: 201}, function(req) {
     return notificationApiService.postNotification(message);
 });
 
+/**
+ * @api {get} /api/2.0/notification/:taskId/:steps
+ * @apiDescription notification get api, used to notify RackHD ipxe initrd downloading finished
+ *  :taskId: active taskId
+ *  :steps: task steps count
+ * @apiName notification-get
+ * @apiGroup notification
+ */
+var notificationIdGet = controller(function(req, res){
+    return notificationApiService.postNotification({
+        taskId: req.swagger.params.taskId.value,
+        progress: {
+            //percentage: "20%",
+            totalSteps: _.parseInt(req.swagger.params.steps.value) || 2,
+            currentStep: 2,
+            description: "iPXE initrd download done, starting initiating installer"
```

Comment:
Although this API by now is only used by posting initrd notification, I would suggest to make it a more generic API. By the URL ```/api/2.0/notification/:taskId/:steps``` has no indication that this API is magically binded to initrd notifcation. 

Consider pass the ```initrd``` information as a parameter to the API so the handler.